azurerm/internal/services/web/resource_arm_function_app.go

```diff
@@ -869,3 +878,14 @@ func flattenFunctionAppSiteCredential(input *web.UserProperties) []interface{} {
 
 	return append(results, result)
 }
+
+func getAccountName(storageConnectionString string) string {
```

Comment:
in Golang functions are shared across the entire package - as such this method would probably be better named as:

```suggestion
func getAccountNameFromStorageConnectionString(storageConnectionString string) string {
```